src/web/file.rs

```diff
@@ -27,9 +27,10 @@ impl File {
             .headers
             .set(ContentType(self.0.mime.parse().unwrap()));
         response.headers.set(CacheControl(cache));
-        response
-            .headers
-            .set(LastModified(HttpDate(time::at(self.0.date_updated))));
+        // FIXME: This is so horrible
+        response.headers.set(LastModified(HttpDate(
+            time::strptime(&self.0.date_updated.format("%+").to_string(), "%Y-%m-%dT%T").unwrap(),
+        )));
```

Comment:
Can you add a round trip test for this?

Reply:
That conversion is by nature lossy because `time` has a lower resolution than `chrono`, but it also probably doesn't matter to terribly much because it doesn't do a whole lot

Reply:
I disagree, unless we test it we don't know that the string formats match. It should be a simple test anyway.

> time has a lower resolution than chrono

That shouldn't affect it? The time starts as `1234.456` or whatever, gets a few zeros added at the end, then the zeros are removed again.